Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -1766,6 +1767,201 @@ def modify_subnet_attribute_command(client: BotoClient, args: Dict[str, Any]) ->
+        """
+        kwargs = {}
+        data = []
+        if args.get("filters"):
+            kwargs.update({"Filters": parse_filter_field(args.get("filters"))})
+        if args.get("subnet_ids"):
+            kwargs.update({"SubnetIds": parse_resource_ids(args.get("subnet_ids"))})
+
+        response = client.describe_subnets(**kwargs)
+
+        if len(response["Subnets"]) == 0:
+            return CommandResults(readable_output="No subnets were found.")
+        for i, subnet in enumerate(response["Subnets"]):
+            data.append(
+                {
+                    "AvailabilityZone": subnet["AvailabilityZone"],
+                    "AvailableIpAddressCount": subnet["AvailableIpAddressCount"],
+                    "CidrBlock": subnet.get("CidrBlock", ""),
+                    "DefaultForAz": subnet["DefaultForAz"],
+                    "State": subnet["State"],
+                    "SubnetId": subnet["SubnetId"],
+                    "VpcId": subnet["VpcId"],
+                    "Region": args["region"],
+                }
+            )
+
+            if "Tags" in subnet:
+                for tag in subnet["Tags"]:
+                    data[i].update({tag["Key"]: tag["Value"]})
+
+        try:
+            output = json.dumps(response["Subnets"], cls=DatetimeEncoder)
+            raw = json.loads(output)
+            raw[0].update({"Region": args["region"]})
+        except ValueError as e:
+            raise DemistoException(f"Could not decode/encode the raw response - {e}")
+        return CommandResults(
+            outputs=raw,
+            outputs_prefix="AWS.EC2.Subnets",
+            outputs_key_field="SubnetId",
+            readable_output=tableToMarkdown(
+                "AWS EC2 Subnets",
+                data,
+                headers=[
+                    "SubnetId",
+                    "AvailabilityZone",
+                    "AvailableIpAddressCount",  # noqa: E501
+                    "CidrBlock",
+                    "DefaultForAz",
+                    "State",
+                    "VpcId",
+                    "Region",
+                ],
+                removeNull=True,
+            ),
+            raw_response=response,
+        )
+
+    @staticmethod
+    def describe_ipam_resource_discoveries_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """Describes one or more IPAM resource discoveries.
+        Args:
+            client (BotoClient): The boto3 client for EC2 service
+            args (Dict[str, Any]): Command arguments including filters, max results, next token, and IPAM resource discovery IDs
+
+        Returns:
+            CommandResults: Results of the operation with IPAM resource discovery information
+        """
+        kwargs = {
+            "Filters": parse_filter_field(args.get("filters")),
+            "MaxResults": int(args.get("limit", DEFAULT_LIMIT_VALUE)),
+            "NextToken": args.get("next_token"),
```

Comment:
Did we check this?

Reply:
After talking to Moish, decided to not test those arguments in the meantime